rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/NodeSource.java

```diff
@@ -59,36 +59,46 @@
     /** if the node source is not deployed, it has no node */
     private NodeSourceStatus nodeSourceStatus;
 
-    NodeSource(String sourceName, String sourceDescription, String nodeSourceAdmin, String nodeSourceStatus) {
-        this.sourceDescription = sourceDescription;
+    private String eventType;
+
+    NodeSource(String sourceName, String sourceDescription, String nodeSourceAdmin, String nodeSourceStatus,
+            String eventType) {
         this.sourceName = sourceName;
+        this.sourceDescription = sourceDescription;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)